conf/timeseries-128shards-source.conf

```diff
@@ -1,4 +1,21 @@
     dataset = "prometheus"
+
+    definition {
+      partition-columns = ["tags:map"]
+      data-columns = ["timestamp:ts", "value:double"]
+      row-key-columns = [ "timestamp" ]
```

Comment:
Can we have a default for this so it doesn't need to be included every time?

Reply:
I suggest that we not have defaults for dataset configuration. This typically does not change and need to be carefully set at the beginning. Defaults are error prone, and do not help the person understand every configuration fully.